nilearn/surface/testing.py

```diff
@@ -0,0 +1,26 @@
+# Helper functions for testing surface code
+
+import numpy as np
+from scipy.spatial import Delaunay
+
+
+def _generate_surf():
```

Comment:
These functions should probably not be private (though the module might be). They should also have a tiny docstring.